tests/foreman/cli/test_fact.py

```diff
@@ -61,3 +61,56 @@ def test_negative_list_by_name(module_target_sat):
     assert (
         module_target_sat.cli.Fact().list(options={'search': f'fact={gen_string("alpha")}'}) == []
     )
+
+
+def test_positive_custom_facts(module_target_sat):
+    """Create custom facts and verify they are updated on Satellite
+
+    :id: ae0b5574-cc8b-4f0c-ba7b-c6f480c08e06
+
+    :steps:
+        1. Create few custom facts
+        2. Verify on Satellite
+
+    :expectedresults: Custom facts are created and uploaded successfully.
+    """
+    ptable = module_target_sat.cli_factory.make_partition_table()
+    os = module_target_sat.cli_factory.make_os()
+    module_target_sat.cli.PartitionTable.add_operating_system(
+        {'name': ptable['name'], 'operatingsystem': os['title']}
+    )
+    ptable = module_target_sat.cli.PartitionTable.info({'name': ptable['name']})
+    assert os['title'] in ptable['operating-systems']
+
+
+@pytest.mark.parametrize(
+    'fact', ['uptime', 'os::family', 'uptime_seconds', 'memorysize', 'ipaddress']
+)
+def test_positive_verify_imported_values(fact, module_target_sat):
+    """Verify imported values
+
+    :id: 64f19639-81d2-47e2-a10d-8456b07b4828
+
+    :steps:
+        1. upload all fact values on Satellite
+        2. Verify on Satellite
+
+    :expectedresults: All facts values are verified.
+    """
+    facts = module_target_sat.cli.Fact().list(options={'search': f'fact={fact}'})
+    assert facts[0]['fact'] == fact
```

Comment:
This test duplicates `test_positive_list_by_name` in the same module. Please consider not adding it.

Reply:
I will be removing this test and extending https://github.com/SatelliteQE/robottelo/pull/14933/files : test_positive_upload_host_facts for more assertions.